### PR TITLE
fix in my-expense category icon

### PR DESCRIPTION
**Pull request by madhavmansuriya40**
old app
Screenshot 2021-03-05 at 3 59 43 PM

new app without changes
Screenshot 2021-03-05 at 3 59 57 PM

new app with changes
Screenshot 2021-03-05 at 5 30 57 PM

check 660 rs expense, taxi category

### CODE REVIEW

File: src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
       [ngClass]="{'my-expenses-card--body__disabled': isDraft && selectionMode && baseState !== 'draft', 'my-expenses-card--body__selected': isSelected}">
       <div class="my-expenses-card--body-internal">
         <div class="my-expenses-card--icon-container" *ngIf="!selectionMode">
-          <app-fy-category-icon [category]="expense.tx_fyle_category"></app-fy-category-icon>
+          <app-fy-category-icon [category]="(expense.tx_extracted_data && expense.tx_extracted_data.category && expense.tx_fyle_category === 'Unspecified' ? (expense.tx_extracted_data.category) : expense.tx_fyle_category)"></app-fy-category-icon>
```
Comment (madhavmansuriya40) [inline src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html:31]: the code snippet is taken from old mobile app

Comment (iamtarun1993) [reply src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html:31]: Screenshot sir, before and after changes??

**Review by iamtarun1993 — APPROVED** [review]

File: src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html (hunk 1 above)
Comment (abhishek1234321) [reply src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html:31]: yes, screenshot please @madhavmansuriya40

**Review by abhishek1234321 — CHANGES_REQUESTED** [review]

File: src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html (hunk 1 above)
Comment (madhavmansuriya40) [reply src/app/fyle/my-expenses/my-expenses-card/my-expenses-card.component.html:31]: sir have attached screenshots in conversations

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**